utils/src/main/scala/za/co/absa/enceladus/utils/transformations/ArrayTransformations.scala

```diff
@@ -22,8 +22,11 @@ import org.apache.spark.sql.types._
 import org.apache.spark.sql.api.java.UDF2
 import org.slf4j.LoggerFactory
 import org.apache.spark.storage.StorageLevel
+import za.co.absa.enceladus.utils.implicits.OptionImplicits.OptionEnhancements
+import za.co.absa.enceladus.utils.udf.UDFNames
 import za.co.absa.spark.commons.implicits.StructTypeImplicits.StructTypeEnhancementsArrays
 
+//TODO refactor to use spark commons and generally better code
```

Comment:
There are some forgotten TODOs. If we cannot afford to remove those or solve those at the moment, perhaps they could reference some new/existing issues in the project.